src/database/sql/audio.py

```diff
@@ -390,7 +421,8 @@ def get_happy(self, audio_id: str) -> str:
         query = """
             SELECT happy, happy_confidence, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.